src/main/java/org/simlar/simlarserver/controllers/SimlarErrorController.java

```diff
@@ -50,6 +50,9 @@
         return ResponseEntity.status(status).contentType(MediaType.APPLICATION_XML).body(new XmlError(response.getId(), response.getMessage()));
     }
 
+    // TODO: fix once spring introduces a proper solution
+    // see: https://github.com/spring-projects/spring-boot/issues/19844
+    @SuppressWarnings({"deprecation", "RedundantSuppression"})
```

Comment:
The issue was closed because for them this is the best solution without breaking backward-compatibility of the API.